python/cudf/cudf/utils/ioutils.py

```diff
@@ -873,11 +874,9 @@ def is_file_like(obj):
 
 
 def _is_local_filesystem(fs):
-    local_filesystem_protocols = ["file"]
 
-    if hasattr(fs, "protocol"):
-        if fs.protocol in local_filesystem_protocols:
-            return True
+    if isinstance(fs, fsspec.implementations.local.LocalFileSystem):
```

Comment:
Why not return directly?
```suggestion
  return isinstance(fs, fsspec.implementations.local.LocalFileSystem)
```

Reply:
Agreed! Thanks for the suggestion.